### ISSUE TITLE
Parser doesn't recognize all links in message

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#18]
```
Вероятно, баг. Если в тексте встречается 
больше одной ссылки, последние 
никак не подсвечиваются. Срабатывает лишь 
для первой/
```

Original issue reported on code.google.com by `Dr.AKULA...@gmail.com` on 19 Dec 2009 at 5:51

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 19 Dec 2009 at 5:56
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Closed by 56229bd232 revision.

I hate GLib Regex -__\
```

Original comment by `tro...@gmail.com` on 20 Dec 2009 at 7:53
- Changed state: **Fixed**
